minisat/utils/StreamBuffer.h

```diff
@@ -24,7 +24,9 @@ OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWA
 #include <stdlib.h>
 #include <stdio.h>
 
+#ifndef NO_ZLIB
```

Comment:
I wonder whether you still need this, since commit "Move StreamBuffer to separate file (such that only main depends on zlib)" has been added.